tools/lsp/ui/views/property-view.slint

```diff
@@ -49,29 +49,31 @@ component ResetButton inherits Button {
 component NameLabel inherits HorizontalLayout {
     in property <ElementInformation> element-information;
     in property <PropertyInformation> property-information;
-
+    in property <bool> debug;
     horizontal-stretch: 0;
-
-    Rectangle {
-        width: EditorSpaceSettings.group-indent;
+    if debug: VerticalLayout {
+        alignment: center;
+        Rectangle {
+            background: magenta;
+            width: 2rem;
+            height: 1px;
+        }
     }
-
     BodyText {
         min-width: EditorSizeSettings.min-prefix-text-width;
-        min-height: 2rem;
 
         text: root.property-information.name;
 
         font-size: 1rem;
         font-weight: property-information.value.code != "" ? EditorFontSettings.bold-font-weight : EditorFontSettings.light-font-weight;
+        font-italic: property-information.value.code != "" ? false : true;
```

Comment:
This is `property-information.value.code == ""`

Reply:
I reversed it so the true/false is also reverse.  I think that's what you meant?